libs/core-wrappers/src/lib/Navbar/Navbar.module.css

```diff
@@ -36,3 +36,21 @@
 .nav-icon {
   margin-right: 1rem;
 }
+
+@media(max-width: 768px) {
```

Comment:
Can [Core-Styles `@custom-media`](https://github.com/TACC/Core-Styles/blob/v2.23.1/src/lib/_imports/tools/media-queries.css) be used? [Reference.](https://github.com/TACC/tup-ui/tree/v1.1.6/apps/tup-ui/src/styles#in-component-styles-import-from-core-styles)

Reply:
This works. Done.